diff --git a/envrc.el b/envrc.el
index dd1cfdf..5ec2c44 100644
--- a/envrc.el
+++ b/envrc.el
@@ -394,6 +394,7 @@ also appear in PAIRS."
               (eshell-set-path path)
             (setq-local eshell-path-env path)))
         (when-let* ((info-path (getenv-internal "INFOPATH" env)))
+          (require 'info)
           (setq-local Info-directory-list
                       (append (seq-filter #'identity (parse-colon-path info-path))
                               (when (boundp 'Info-directory-list)
